src/EditorFeatures/Test/Diagnostics/DiagnosticStateTests.cs

```diff
@@ -10,7 +10,7 @@
 using System.Threading.Tasks;
 using Microsoft.CodeAnalysis.CSharp;
 using Microsoft.CodeAnalysis.Diagnostics;
-using Microsoft.CodeAnalysis.Diagnostics.EngineV1;
+using Microsoft.CodeAnalysis.Diagnostics.EngineV2;
```

Comment:
you can just delete this file, I already added v2 version of same test.

DiagnosticDataSerializerTests.cs